src/layout/Header.tsx

```diff
@@ -31,6 +31,9 @@ const Header: FC = () => (
                     Fares Data Build Tool
                 </a>
             </div>
+            <a href="/signIn" id="sign_in_link" className="govuk-header__link signIn govuk-heading-s ">
```

Comment:
I'd get rid of this for now since it doesn't do anything and it makes the header look a bit odd